### PR TITLE
Added "Python Microservice framework"

### PR DESCRIPTION
**Pull request by avara1986**
See more info in:
* https://github.com/python-microservices/
* https://github.com/python-microservices/pyms/
* https://github.com/python-microservices/microservices-scaffold/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,8 +117,7 @@ A curated list of awesome Python frameworks, libraries and software.
 * [MorvanZhou/tutorials](https://github.com/MorvanZhou/tutorials) - 机器学习相关教程
 * [eliangcs/http-prompt](https://github.com/eliangcs/http-prompt) - HTTPie + prompt_toolkit = an interactive command-line HTTP client featuring autocomplete and syntax highlighting
 * [coursera-dl/coursera-dl](https://github.com/coursera-dl/coursera-dl) - Script for downloading Coursera.org videos and naming them.
-* [MorvanZhou/tutorials](https://github.com/MorvanZhou/tutorials) - 机器学习相关教程
```
Comment (ccding) [inline README.md]: why are you removing this one?

File: README.md [hunk 2]
```diff
@@ -117,8 +117,7 @@ A curated list of awesome Python frameworks, libraries and software.
...
-* [falconry/falcon](https://github.com/falconry/falcon) - The minimalist web API framework for building incredibly fast, reliable, and scalable microservices, proxies, and app backends in Python.
+* [falconry/falcon](https://github.com/falconry/falcon) - Too much magic makes your apps hard to debug and hard to scale. Falcon is the minimalist web framework for building incredibly fast, reliable, and scalable microservices, proxies, and app backends.
```
Comment (ccding) [inline README.md]: Just to confirm, does "added" mean updating the description?

File: README.md (hunk 1 above)
Comment (avara1986) [reply README.md]: Hi! Merge request conflict. In uhub/awesome-python was moved to line 117

File: README.md (hunk 2 above)
Comment (avara1986) [reply README.md]: Merge request conflict. In uhub/awesome-python was updated the description

**Comment by avara1986** [comment]
Why is the reason to close the PR? is something worng @ccding ? Thanks!

**Pull request closed without merging**